config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,23 @@
+# TODO: what happens if the ENVs are missing?
+# TODO: is ENV safe way to handle these credentials?
+CarrierWave.configure do |config|
+  # required settings
+  config.fog_provider = 'fog/aws'
+  config.fog_directory  = 'article-header-images'
+
+  # optional settings
+  config.fog_public     = true # defaults to true
+  config.fog_attributes = { 'Cache-Control' => "max-age=#{365.day.to_i}" } # defaults to {}
```

Comment:
maybe use [Cache-Control: immutable][0]?

[0]:https://hacks.mozilla.org/2017/01/using-immutable-caching-to-speed-up-the-web/